__tests__/bank.ts

```diff
@@ -1,11 +1,29 @@
-import { Bank , Reactor, App, Timer, Triggers, Args, Present, OutPort, InPort, TimeValue } from "../src/core/internal";
+import { Bank , Reactor, App, Timer, Triggers, Args, Present, OutPort, InPort, TimeValue, OutMultiPort, Port } from "../src/core/internal";
 
 class Periodic extends Reactor {
     
     t: Timer = new Timer(this, 0, TimeValue.sec(1));
     o: OutPort<number> = new OutPort(this)
     constructor(parent: Reactor) {
         super(parent)
+        let writer = this.writable(this.o);
+        this.addReaction(
+            new Triggers(this.t),
+            new Args(this.t),
+            function (this) {
+                console.log(this.getBankIndex());
+            }
+        );
+    }
+}
+
+class MultiPeriodic extends Reactor {
+    
+    t: Timer = new Timer(this, 0, TimeValue.sec(1));
+    o: OutMultiPort<number> = new OutMultiPort(this, 2)
+    constructor(parent: Reactor) {
+        super(parent)
+        let allWriter = this.allWritable(this.o);
```

Comment:
Looks like this line isn't necessary?

Reply:
Yes, you are right. It escaped my notice.